ext/reflection/php_reflection.stub.php

```diff
@@ -665,3 +671,27 @@ private function __clone(): void {}
 
     private function __construct() {}
 }
+
+final class ReflectionEnum extends ReflectionClass
+{
+    /** @return bool */
+    public function hasCase(string $name) {}
```

Comment:
The reason that existing reflection classes don't have return types is backwards compatibility, e.g. `SomeAlternateReflectionClass->getMethod()` may return a different return type since PHP 5.

Backward compatibility is not a concern for `final class ReflectionEnum` because it can't be subclassed. If we add a subclass of ReflectionEnum it might matter, but adding more methods seems more likely

```suggestion
    public function hasCase(string $name): bool {}
```